### PR TITLE
Updated Pselect's help to hopefully be clearer and more comprehensive.

### PR DESCRIPTION
**Pull request by cianoc**
Just general rewrites, plus the new method wasn't properly documented for Pselect.

### CODE REVIEW

File: HelpSource/Classes/Pselect.schelp [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
-related:: Classes/Pcollect, Classes/Preject
+related:: Classes/Pcollect, Classes/Preject, Classes/Collection#-select
 
 description::
-Returns values for which the function returns true. The value is passed to the function.
+This pattern will filter the source pattern using the supplied function strong::func::.
 
+Values from the source pattern will be passed to strong::func::. Pselect will only return that value if the strong::func:: returns true.
+
+This is the pattern libraries equivalent of link::Classes/Collection#-select#select::.
```
Comment (brianlheim) [inline HelpSource/Classes/Pselect.schelp]: -> library's

File: HelpSource/Classes/Pselect.schelp [hunk 2]
```diff
@@ -1,19 +1,23 @@
 class:: Pselect
-summary:: Select values from a pattern
+summary:: Filters values from return by a source pattern.
```
Comment (brianlheim) [inline HelpSource/Classes/Pselect.schelp]: -> Filters values returned by a ...

**Comment by cianoc** [comment]
@LFSaw those two functions aren't private, so I don't think that would be the right thing to do.

**Comment by telephon** [comment]
It would be god to have a different tag for methods that aren't private but which we want to hide.

`hide::`

**Comment by cianoc** [comment]
I think hide would be the wrong tag, as really we want this to appear as inherited instance methods, even though they've been overloaded. So I guess what you want is a way of tagging this as a method belonging to Pattern.

By which I mean it's totally fine to use these methods, but it's better that people read about them in the context of Pattern, than here.

**Comment by LFSaw** [comment]
since we agree on the values of these but start to talk about higher-level decisions, I merged.

**Pull request merged**